src/modules/universal-discovery/components/sort-switcher/sort.switcher.js

```diff
@@ -4,10 +4,14 @@ import PropTypes from 'prop-types';
 import MenuButton from '../menu-button/menu.button';
 
 import { createCssClassNames } from '../../../common/helpers/css.class.names';
-import { SortingContext, SORTING_OPTIONS } from '../../universal.discovery.module';
+import { SortingContext, SortOrderContext, SORTING_OPTIONS } from '../../universal.discovery.module';
 
 const SortSwitcher = ({ isDisabled }) => {
     const [sorting, setSorting] = useContext(SortingContext);
+    const [sortOrder, setSortOrder] = useContext(SortOrderContext);
+
+    const ascendingOrder = 'ascending';
```

Comment:
personally I would move this kind of consts outside of component and write them in uppercase like `const ASCENDING_ORDER = 'ascending'` to distinct them from other variables, question for @dew326 what he thinks about it